state/sshconnrequests.go

```diff
@@ -150,3 +152,17 @@ func (st *State) WatchSSHConnRequest(unitName string) StringsWatcher {
 		},
 	})
 }
+
+// CleanupExpiredRecord removes the expired record from the collection.
```

Comment:
Comment has wrong casing for the method name and does not describe what this method does.